src/Microsoft.AspNet.Mvc.Razor/RazorViewFactory.cs

```diff
@@ -21,11 +23,14 @@ public class RazorViewFactory : IRazorViewFactory
         /// <param name="pageActivator">The <see cref="IRazorPageActivator"/> used to activate pages.</param>
         /// <param name="viewStartProvider">The <see cref="IViewStartProvider"/> used for discovery of _ViewStart
         /// pages</param>
+        /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to log data.</param>
         public RazorViewFactory(IRazorPageActivator pageActivator,
-                                IViewStartProvider viewStartProvider)
+                                IViewStartProvider viewStartProvider,
+                                ILoggerFactory loggerFactory)
```

Comment:
Send mail to the eng team, and share the results with the team. This keeps coming up.